src/app/store/store.component.ts

```diff
@@ -10,12 +12,18 @@ import { Product } from 'app/core/store/product.model';
 export class StoreComponent implements OnInit {
   products: Array<Product>;
 
-  constructor(private storeService: StoreService) {}
+  constructor(private storeService: StoreService,
+    private router: Router,
+    private route: ActivatedRoute) {}
```

Comment:
also very  minor but if we're splitting into multiple lines, i like putting each one on it's own line (so like move store service down), just so they all line up and are easy to read.